lib/telemetry/armon_metrics.go

```diff
@@ -196,9 +196,54 @@ type TelemetryConfig struct {
 	StatsiteAddr string `json:"statsite_address,omitempty" mapstructure:"statsite_address"`
 }
 
+// sinkFn takes Config and a hostname and builds a sink to be composed in the FanOutSink
```

Comment:
```suggestion
// sinkFn takes Config and builds a sink to be composed in the FanOutSink
```

Is this the right change or should it actually take a hostname?

Reply:
Yes, your revision is correct. We refactored away the hostname because that label is assigned by go-metrics. The comment is incorrect and should be amended.